summarization/stamp_page_picking/scoring_utils.py

```diff
@@ -0,0 +1,163 @@
+''' Module for scoring utils '''
+
+from sklearn.metrics.pairwise import cosine_similarity
+
+
+class ScoringUtils:
+    SCORE_FOR_TEXT_ONLY_STAMP = 1.0
+    SCORE_FOR_EMBEDDED_STAMP = 5.0
+    SCORE_FOR_VISUAL_STAMP = 10.0
+
+    def __init__(self, stamp_pages, stamp_page_covers, cover_size):
+        self.stamp_pages = stamp_pages
+        self.stamp_page_covers = stamp_page_covers
+        self.cover_size = cover_size
+        self.picked_cover = [0] * self.cover_size
+
+        # sweeping index is initially the maximum or last index possible
+        self.sweeping_index = max(
+            [max(page.media_index, page.sentence_index)
+             for page in self.stamp_pages]
+        )
+
+        self.last_picked_stamp_page_index = -1
+
+        # step size to change sweeping index
+        # TODO: add variable change in step size
+        # based on iteration
+        self.step_size = 1
+        self.iteration_count = 1
+
+        self.individual_score_weights = [
+            1,  # content change score weight
+            0,  # sentiment change score
+            0,  # textual entailment score
```

Comment:
Can you remove (stash these changes somewhere) the metrics (and related methods for the ones) in this PR.

When we add those metrics, it'll be easier to review them then.

Reply:
Okay!